packages/next/src/build/webpack/config/blocks/css/index.ts

```diff
@@ -576,6 +578,8 @@ export const css = curry(async function css(
             // Exclude extensions that webpack handles by default
             exclude: [
               /\.(js|mjs|jsx|ts|tsx)$/,
+              // TODO investigate why this is needed
```

Comment:
this is used to workaround a old bug which maybe solved now , I need to check it

Reply:
I checked using pages-dir and app-dir cases and it seems runs well without this hack, so it seems you can remove the hack now